### DISCUSSION TITLE
WeChat Group QR Code

### DISCUSSION THREAD

**Discussion by cyfyifanchen** [TEN-framework/ten-framework#170] [Announcements]
<div align="center">

<img width="434" height="520" alt="image" src="https://github.com/user-attachments/assets/05ce1d96-e174-405a-a3b3-a25a9f8c2b44" />

</div>

**Comment by Virtuoso461** [comment]
我需要群 可以重新发一下吗

**Reply by cyfyifanchen** [reply]
群人数超过自动加入上限了，可以加我微信，我来你进去。

**Reply by Virtuoso461** [reply]
好的 那我加你
